src/main/java/error/NotValidBonusDuplicatedException.java

```diff
@@ -0,0 +1,4 @@
+package error;
+
+public class NotValidBonusDuplicatedException extends IllegalArgumentException{
+}
```

Comment:
사용자 정의 예외를 만들어서 사용하는 부분이 굉장히 좋은것 같아요!
그런데 사용자 정의 예외 생성자의 매개변수로 예외메세지를 넘겨주고, try-catch 로 잡는 부분에서 메세지를 반환하도록 하는건 어떤가요?
ErrorType 클래스에 예외 메세지를 잘 정리 해 주셔서 함께 사용하면 정의한 예외와 에러 메세지가 연관지어져 있어서 조금 더 의도가 명확 해 진다고 생각해요!

```suggestion
public class NotValidBonusDuplicatedException extends IllegalArgumentException {
    public NotValidBonusDuplicatedException(String message) {
        super(message);
    }
}


// 예외를 try-catch 하는 부분
try {
    throw new NotValidBonusDuplicatedException(ErrorType.INVALID_BONUS_NUMBER_DUPLICATED.getText());
} catch (NotValidBonusDuplicatedException ex) {
    System.out.println( ex.getMessage());
}

```